provisioning/baremetal_pod.go

```diff
@@ -584,6 +585,17 @@ func newMetal3PodTemplateSpec(images *Images, config *metal3iov1alpha1.Provision
 		},
 	}
 
+	for _, cont := range containers {
+		// these are "default" values to pass openshift e2e tests.
+		// to set specific values, define them in the container definition.
+		if _, ok := cont.Resources.Requests[corev1.ResourceCPU]; !ok {
+			cont.Resources.Requests[corev1.ResourceCPU] = resource.MustParse("10m")
+		}
+		if _, ok := cont.Resources.Requests[corev1.ResourceMemory]; !ok {
+			cont.Resources.Requests[corev1.ResourceMemory] = resource.MustParse("50Mi")
+		}
+	}
```

Comment:
How those values have been calculated ? Could make sense to try using the formula suggested in the conventions doc for determining them more accurately?

Reply:
I set the same as this: https://github.com/openshift/cluster-baremetal-operator/blob/master/config/cluster-baremetal-operator/cluster-baremetal-operator.yaml#L37-L40

Reply:
any idea what a better default would be?